fbw-a32nx/src/wasm/fbw_a320/src/interface/SimConnectInterface.cpp

```diff
@@ -2075,27 +2078,37 @@ void SimConnectInterface::processEventWithOneParam(const DWORD eventId, const DW
     }
 
     // TODO Unsync EFIS baro
+    case Events::KOHLSMANN_INC:
```

Comment:
You should check the index in param0 of this event to ensure it's targeted to your altimeter and pass it on if not. Same for the DEC event below.

Reply:
Same here